eng/Versions.props

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.5.0</VersionPrefix>
+    <VersionPrefix>17.5.0</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
```suggestion
    <VersionPrefix>17.5.0</VersionPrefix>
    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Reply:
I want us to format this badly, because it creates a merge conflict on the merge back to `main` where we've changed prefix to `17.6.0`. In the past, good formatting has caused Git to cleanly merge and accidentally produced a "17.x.0" build that was actually pre-preview1 content.